### ISSUE TITLE
How to prepare target heatmaps?

### ISSUE THREAD

**Issue by mukeshnarendran7** [yangsenius/TransPose#29]
I want to use the pre-trained model and fine-tune it for another application but I am not able to find the heatmaps preparation code reference? Is it similar to taking an image an converting the (x,y) co-ordinates to heatmaps like for CNN's pose estimation problem? The model output is of (48,64) but my input images are 256, 192. A reference will be helpful. Thanks

**Comment by yangsenius** [comment]
Please refer to https://github.com/yangsenius/TransPose/blob/dab9007b6f61c9c8dce04d61669a04922bbcd148/lib/dataset/JointsDataset.py#L239

**Comment by mukeshnarendran7** [comment]
Thanks. I have another query on how the final layer is transformed while testing the mpii dataset. In the paper, it is mentioned that you need to use a d*16 fully connected layer in place of the final layer with a pre-trained network.  If you could share a code example would be great because I am a bit confused with the output as they are not heatmaps then.  How would you compare to the  target?

**Comment by yangsenius** [comment]
1. Yes, you're right.
2. JointMSEloss considers more detailed implementation, such as the visibility of the keypoints. Essentially, they are the same loss function. But I suggest to use the JointMSE loss.
